app/models/report_generators/system_performance/fy2019/measure_six.rb

```diff
@@ -237,22 +237,21 @@ def measure_6_a_and_b
-      @support[:sixab_g3][:support] = support_for(answer: :sixab_g3, data: project_exit_counts[:g_366_730_days][:es][:support])
-      @support[:sixab_g4][:support] = support_for(answer: :sixab_g4, data: project_exit_counts[:g_366_730_days][:th][:support])
-      @support[:sixab_g5][:support] = support_for(answer: :sixab_g5, data: project_exit_counts[:g_366_730_days][:sh][:support])
-      @support[:sixab_g6][:support] = support_for(answer: :sixab_g6, data: project_exit_counts[:g_366_730_days][:ph][:support])
-
+      @support[:sixab_c2][:support] = support_for(answer: :sixab_c2, data: project_exit_counts[:c_0_180_days][:so][:support], personal_ids: client_personal_ids)
+      @support[:sixab_c3][:support] = support_for(answer: :sixab_c3, data: project_exit_counts[:c_0_180_days][:es][:support], personal_ids: client_personal_ids)
+      @support[:sixab_c4][:support] = support_for(answer: :sixab_c4, data: project_exit_counts[:c_0_180_days][:th][:support], personal_ids: client_personal_ids)
+      @support[:sixab_c5][:support] = support_for(answer: :sixab_c5, data: project_exit_counts[:c_0_180_days][:sh][:support], personal_ids: client_personal_ids)
+      @support[:sixab_c6][:support] = support_for(answer: :sixab_c6, data: project_exit_counts[:c_0_180_days][:ph][:support], personal_ids: client_personal_ids)
+      @support[:sixab_e2][:support] = support_for(answer: :sixab_e2, data: project_exit_counts[:e_181_365_days][:so][:support], personal_ids: client_personal_ids)
+      @support[:sixab_e3][:support] = support_for(answer: :sixab_e3, data: project_exit_counts[:e_181_365_days][:es][:support], personal_ids: client_personal_ids)
+      @support[:sixab_e4][:support] = support_for(answer: :sixab_e4, data: project_exit_counts[:e_181_365_days][:th][:support], personal_ids: client_personal_ids)
+      @support[:sixab_e5][:support] = support_for(answer: :sixab_e5, data: project_exit_counts[:e_181_365_days][:sh][:support], personal_ids: client_personal_ids)
+      @support[:sixab_e6][:support] = support_for(answer: :sixab_e6, data: project_exit_counts[:e_181_365_days][:ph][:support], personal_ids: client_personal_ids)
+      @support[:sixab_g2][:support] = support_for(answer: :sixab_g2, data: project_exit_counts[:g_366_730_days][:so][:support], personal_ids: client_personal_ids)
+      @support[:sixab_g3][:support] = support_for(answer: :sixab_g3, data: project_exit_counts[:g_366_730_days][:es][:support], personal_ids: client_personal_ids)
+      @support[:sixab_g4][:support] = support_for(answer: :sixab_g4, data: project_exit_counts[:g_366_730_days][:th][:support], personal_ids: client_personal_ids)
+      @support[:sixab_g5][:support] = support_for(answer: :sixab_g5, data: project_exit_counts[:g_366_730_days][:sh][:support], personal_ids: client_personal_ids)
+      @support[:sixab_g6][:support] = support_for(answer: :sixab_g6, data: project_exit_counts[:g_366_730_days][:ph][:support], personal_ids: client_personal_ids)
```

Comment:
Same here, I doubt these are all the same people in every bucket.

Reply:
And, I believe the same here -- the support field comes from `initial_exit` which comes from `project_exits_to_ph`